opm/simulators/flow/FIBlackoilModel.hpp

```diff
@@ -48,6 +48,20 @@
 
 namespace Opm
 {
+
+namespace detail
```

Comment:
This is exactly ThreadManager::maxThreads(), why duplicate?

Reply:
Had forgotten about that, will remove!